x-pack/plugins/security_solution/public/cloud_security_posture/links.ts

```diff
@@ -4,70 +4,56 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import {
-  getSecuritySolutionDashboardLinks,
-  getSecuritySolutionManageLinks,
-  getSecuritySolutionRootLinks,
-} from '@kbn/cloud-security-posture-plugin/public';
+import { getSecuritySolutionLink } from '@kbn/cloud-security-posture-plugin/public';
 import { i18n } from '@kbn/i18n';
-import { SecurityPageName } from '../../common/constants';
+import { SecurityPageName, SERVER_APP_ID } from '../../common/constants';
 import cloudSecurityPostureDashboardImage from '../common/images/cloud_security_posture_dashboard_page.png';
 import type { LinkCategories, LinkItem } from '../common/links/types';
 import { IconExceptionLists } from '../management/icons/exception_lists';
 
 const commonLinkProperties: Partial<LinkItem> = {
   hideTimeline: true,
-  experimentalKey: 'cloudSecurityPostureNavigation',
+  capabilities: [`${SERVER_APP_ID}.show`],
```

Comment:
could you clarify this change?

Reply:
experimental key was removed to show our nav in security solution by default. capabilities were added to only show our nav options to users with this capability